src/main/resources/application-local.yaml

```diff
@@ -15,7 +15,7 @@ spring:
             enable: true
 
 keycloak:
-  base-url: "http://0.0.0.0:9080"
+  base-url: "http://192.168.0.79:9080"
```

Comment:
Due to the state that the mobile app accesses the Keycloak instance using the local ip of the device, it had to be changed.
The issuer that will be validated differs otherwise.


We probably should add a separate config that is ignored by git.